setup.cfg

```
@@ -1,10 +1,8 @@
 [tool:pytest]
-addopts = --cov=aiohttp_devtools --cov-report xml --cov-report html --cov-report term
-markers = boxed: mark tests as boxed
+addopts = --cov=aiohttp_devtools --cov-report xml --cov-report html --cov-report term --forked
```

```suggestion
addopts = --cov=aiohttp_devtools --cov-report xml --cov-report html --cov-report term
```
(so that only marked tests are isolated)

---

When I do this, it doesn't seem to fork any of the tests. Is there something else I'm meant to add to the config?

---

No, it should just work. But I'm not sure if it'll be visible in the report at all.

---

Well, the CI is failing as it does on my machine, which I believe is because the tests are not forking.

---

I've just verified that pytest-forked tests this case: https://github.com/pytest-dev/pytest-forked/blob/f10fb75/testing/test_boxed.py#L22-L34. So I'm pretty sure it should work.

---

Maybe it's caused by other tests not being marked? (since `--forked` would've marked _every_ test regardless of the decorator)

---

Yep, that was it. Curious how they ever worked in the past...